client/src/pages/trainer/course/detail/[id].tsx

```diff
@@ -87,8 +88,8 @@ const CourseDetails: React.FC = () => {
               <div className="flex justify-end pr-6">
                 <Button text={'Add Course Content'} color={'#325184'}></Button>
               </div>
-              <div className="flex justify-center items-center h-auto w-auto font font-medium text-xl">
-                Content for Tab {activeTab}
+              <div className="flex h-auto w-auto font font-medium">
+                {activeTab === 1 ? <CourseSetting /> : null}
```

Comment:
As much as possible. optimize the code by excluding empty condition. 

```suggestion
               {activeTab === 1 && <CourseSetting />}
```